CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [2.8.0] - January 18, 2022
+
+### New Features
+- Add a set of new APIs for overriding and managing user-level flag, experiment and delivery rule decisions. These methods can be used for QA and automated testing purposes. They are an extension of the ReactSDKClient interface ([#133](https://github.com/optimizely/react-sdk/pull/133)):
+	- setForcedDecision
+	- getForcedDecision
+	- removeForcedDecision
+	- removeAllForcedDecisions
+- Updated `useDecision` hook to auto-update and reflect changes when forced decisions are set and removed.
+- For details, refer to our documentation pages: [ReactSDKClient](https://docs.developers.optimizely.com/full-stack/v4.0/docs/reactsdkclient) and [useDecision hook](https://docs.developers.optimizely.com/full-stack/v4.0/docs/usedecision-react).
```

Comment:
These docs do not include FD. I believe, contents will be updated with the same doc links.

Reply:
Yes! I am holding off merging and releasing at least till we have the page skeletons final. This is expected to be updated a little bit